src/styles/index.scss

```diff
@@ -4,0 +1,7 @@
+@import '/src/styles/blocks/variables';
+@import '/src/styles/blocks/page';
```

Comment:
Ensure that the import path for the page styles is correct and relative to the SCSS file location. The current path '/src/styles/blocks/page' should be './blocks/page' to align with the project structure.